Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderPrivate.h

```diff
@@ -27,6 +27,7 @@
 
 #if PLATFORM(COCOA) && ENABLE(GPU_PROCESS) && ENABLE(MEDIA_STREAM)
 
+#include "GPUProcessConnection.h"
```

Comment:
What happened here?

Reply:
Build fix since adding files to Source.txt sometimes uncovers missing header imports.